WalletWasabi/WabiSabi/Backend/DoSPrevention/Warden.cs

```diff
@@ -89,4 +89,22 @@ public override Task StopAsync(CancellationToken cancellationToken)
 		OffendersToSaveChannel.Writer.Complete();
 		return base.StopAsync(cancellationToken);
 	}
+
+	public void BanDescendant(object? sender, Block block)
```

Comment:
Feels like this method could be inside `Prison` itself.

Reply:
This is a level of abstraction above `Prision`. I was not sure where to this belong to because there where two candidate places: `Warden` and `WabisabiCoordinator`. In #11372 the method `BanDoubleSpenders` belong clearly to the `WabisabiCoordinator` because it needs to know about the `Prison` and `Arena` but this method doesn't requires access to `Arena`.  

OTOH it could be good to have both methods  `BanDescendant` and `BanDoubleSpenders` together in the same class ( `WabisabiCoordinator`)

Reply:
I moved it to `WabiSabiCoordinator` so all banning logic is there.